src/invocation/ClusterService.ts

```diff
@@ -78,7 +79,7 @@ export class ClusterService implements Cluster {
      * @return The member that was found, or undefined if not found.
      */
     getMember(uuid: string): MemberImpl | undefined {
-        assertNotNull(uuid);
+        assertNotNullCond(uuid);
```

Comment:
Same as below, I have checked all code paths that use this. It is not possible to pass a null UUID here